src/service/randomNumberInRange.js

```diff
@@ -0,0 +1,44 @@
+// min should be a minimal number, max should be a maximal number, forbiddenNumbersArray should be an array containing all the numbers that should not be generated
+
+export const randomNumberInRange = (min, max, forbiddenNumbersArray = []) => {
+    
+    if (typeof min === 'number' && typeof max === 'number' && Array.isArray(forbiddenNumbersArray) && min <= max) {
+
+        // check if not every possible number is forbidden
+        if (max - min + 1 === forbiddenNumbersArray.length) {
+
+            const allPossibleOutComes = [];
+
+            for (let i = min; i <= max; i++) {
+                allPossibleOutComes.push(i);
+            }
+
+            const allPossibleOutComesSorted = allPossibleOutComes.sort((a, b) => {return a-b});
+            const forbiddenNumbersArraySorted = forbiddenNumbersArray.sort((a, b) => {return a-b});
+
+            if  (allPossibleOutComesSorted.every((val, index) => val === forbiddenNumbersArraySorted[index])) {
+                throw Error('Invalid arguments!')
+            }
+        }
+        
+        // generate numbers
+
+        const generateNumber = () => {
+            return Math.floor(Math.random() * (max - min) + min); 
+         }
+         
+         let randomNumber = generateNumber();
+
+         // check if a generated number is not included in a forbidden numbers
+     
+         while (forbiddenNumbersArray.includes(randomNumber)) {
```

Comment:
❗IMPORTANT❗ 
[Functionality and time complexity]
if you want to get for example number between 1 and 1'000'000 and make all numbers from 1 to 999'999 forbidden then this will take really long time to execute.

It can be done with less time complexity if you would:
1. create array of all possibilities
2. remove all forbidden numbers from possibilities
3. from this `'allPossibilities - forbiddenNumbers'` array get one element between index `0` and `array.length-1` at random.

It can be achieved easier with `Set` class. Let me know if you need more tips 🧑‍🚀